apps/coscrad-frontend/src/components/dynamic-forms/form-state/types/form-state.ts

```diff
@@ -0,0 +1,2 @@
+// TODO tighten up the types
+export type FormState = Record<string, unknown>;
```

Comment:
Make an effort at this.

Reply:
This may be unnecessary complication. This state \ its type are entirely dynamic. We do not manually manipulate them ever. We're better to rely on automated tests than statics for confidence with this one.